wallet-js/LICENSE_MIT

```diff
@@ -0,0 +1,25 @@
+Copyright (c) 2018 Enzo Cioppettini <ecioppettini@atixlabs.com>
```

Comment:
```suggestion
Copyright (c) 2020 Input Output HK
```

You are the author of it in the crate already, but the copyright is still IOHK's